lang/fr/texts/ingredients-pouvant-etre-issus-de-l-huile-de-palme.list.html

```diff
@@ -1,14 +1,14 @@
 <img id="og_image" src="/images/fr/ingredients_that_may_be_from_palm_oil/ingredients-that-may-be-from-palm-oil.90x90.png" width="90" height="90"/ style="float:left;margin-right:20px;margin-bottom:10px;">
 
-<p>La l�gislation fran�aise n'impose pas aux fabricants de produits alimentaires de pr�ciser s'ils contiennent ou non de l'huile de palme. La mention "huile v�g�tale" dans les ingr�dients est cependant une indication que le produit en contient
-probablement : lorsque des huiles plus on�reuses comme l'huile de tournesol sont utilis�es, les fabricants le pr�cisent le plus souvent. Pour �tre certain de la pr�sence ou de la non pr�sence d'huile de palme, il est
-n�cessaire de contacter le fabricant.</p>
+<p>La l�gislation fran�aise n'impose pas aux fabricants de produits alimentaires de pr�ciser s'ils contiennent ou non de l'huile de palme. La mention "huile v�g�tale" dans les ingr�dients est cependant une indication que le produit en contient

```

Comment:
I think @teolemon disabled builds for PRs because commits in the openfoodfacts/openfoodfacts-server repository and the pull requests from the same repo would cause duplicate builds.